runtimevar/runtimevar.go

```diff
@@ -483,19 +485,16 @@ func bytesDecode(b []byte, obj interface{}) error {
 // DecryptDecode returns a decode function that can be passed to NewDecoder when
 // decoding an encrypted message (https://godoc.org/gocloud.dev/secrets).
 //
-// The decode function by default will decode the input into raw bytes. An
-// optional decoder can be passed in to do further decode operation based on the
-// decrypted message.
+// The decode function default to BytesDecode. An optional decoder can be passed
```

Comment:
Nit: "post defaults to ByteCode. An optional...".

Reply:
Done.